examples/nginx/site-available

```diff
@@ -0,0 +1,52 @@
+# This is an example nginx `sites-available` file to use with maxwell-daemon.
+# Lua scripting is used to communicate with the daemon.
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    location / {
+        # Arbitrate the request's path (defaulting to master).
+        set $maxwell "master";
+        access_by_lua '
```

Comment:
Depending on whether you consider this as the `supported` client for this rollout service. Given nginx is the only client, we might as well test it. It's up to you though.